style.css

```diff
@@ -157,10 +179,87 @@ a:active {
   margin-bottom: 24px;
 }
 
+.buttons {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-evenly;
+}
+
+.detail-button {
+  background-color: var(--greenThemeText);
+  border-radius: 4px;
+  color: #fff;
+  text-align: center;
+  padding: 15px;
+  border: 0;
+  margin-top: 5px;
+  margin-bottom: 15px;
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 24px;
+  display: flex;
+  align-items: center;
+  letter-spacing: 0.03em;
+}
+
 .desktop-hr {
   display: none;
 }
 
+.detail-description {
+  font-family: "Inter", sans-serif;
+  font-size: 16px;
+  font-style: normal;
+  font-weight: 400;
+  line-height: 24px;
+  text-align: left;
+  margin-top: 30px;
+}
+
+.details-heading {
+  font-family: "Inter", sans-serif;
+  font-size: 32px;
+  font-style: normal;
+  font-weight: 600;
+  line-height: 44px;
+  letter-spacing: 0;
+  text-align: left;
+}
+
+#details-modal {
+  position: fixed; /* Stay in place */
+  z-index: 1; /* Sit on top */
+  padding-top: 5px; /* Location of the box */
+  left: 0;
+  top: 0;
+  width: 100%; /* Full width */
+  height: 100%; /* Full height */
+  background-color: rgb(0, 0, 0); /* Fallback color */
+  background-color: rgba(0, 0, 0, 0.4); /* Black w/ opacity */
+}
```

Comment:
[OPTIONAL] It would be better to remove the comments to make your code clean and professional.